src/MemoryPowerModel.h

```diff
@@ -62,7 +62,9 @@ class MemoryPowerModel {
                           double spup_ref_pre_cycles,
                           double clk);
 
-  int64_t total_cycles;
+  int64_t total_cycles = 0;
```

Comment:
Line 65 and 68 break the builds for somewhat older versions of gcc: https://travis-ci.org/ravenrd/DRAMPower/jobs/149783410

Can you please move the initialization of these variables to a different place to fix that?